### DISCUSSION TITLE
Multiobject support in GUI?

### DISCUSSION THREAD

**Discussion by max810** [hkchengrex/XMem#76] [Q&A] [answered]
Hi,

Does GUI support drawing annotations for multiple objects? I specify `--num_objects N` argument, but there seems to be no changes in the UI, i.e. I cannot specify multiple objects when drawing the masks.

When importing the masks, however, all N objects are displayed correctly, but all colors in the editor are treated as one (i.e. they are still erased) and only the color of the first object (red) is available for the drawing.

Am I using the tool wrong or is this something you have not yet implemented?

**Comment by hkchengrex** [comment, marked as answer]
You should be able to use number keys to change the target object.
